### PR TITLE
Show helm release revisions in history tab

### PR DESCRIPTION
**Pull request by divyanshiGupta**
**Fixes:** https://issues.redhat.com/browse/ODC-3158

**Analysis / Root cause:**
Add history tab on helm release details page and show helm release revisions

**Solution Description:**
This PR adds history tab on `HelmReleaseDetailsPage`. Fetches helm release revisions from API endpoint `/api/helm/release/history?ns=${namespace}&name=${name}` and shows it as a list.

**Screen shots / Gifs for design review:**
CC: @openshift/team-ux-review @siamaksade 

Screenshot from 2020-03-20 17-24-18

Updated
Screenshot from 2020-03-23 20-10-07

Updated
Screenshot from 2020-03-27 14-57-58

**Test setup:**
The UI depends on APIs being added by #4580. Need to build the backend on top of that PR in order to test the API.

To-do
- [x] Add unit tests 

### Note - I had to create a new PR due to some local setup issue. Closed #4782 for this one.

### CODE REVIEW

File: frontend/packages/dev-console/src/components/custom-resource-list/CustomResourceList.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+      if (ignore) return;
+
+      setListItems(newListItems || []);
+      setFetched(true);
+
+      if (activeFilters) {
+        const filteredItems = () => {
+          return rowFilterReducer(newListItems, activeFilters);
+        };
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/src/components/custom-resource-list/CustomResourceList.tsx]:
```suggestion
        const filteredItems = rowFilterReducer(newListItems, activeFilters);
```

**Comment by divyanshiGupta** [comment]
@serenamarie125 @parvathyvr by which field the text should be filtered? In helm releases page we are doing it by name but here name is always going to be same.

**Comment by christianvogt** [comment]
@Divyansh42 @parvathyvr I think we agreed that there isn't any value to having name or status filters on this page because there's no value.

Default sort should be revision with the latest revision at the top.

**Comment by christianvogt** [comment]
@divyanshiGupta even if we don't show the filters, the work you did here to make this generic is good for future use cases.

**Comment by divyanshiGupta** [comment]
@christianvogt so the list is by default sorted by revisions now. But I was not sure about the text filter. So you are saying that I should remove text and row filters?

**Comment by christianvogt** [comment]
@Divyansh42 yes but confirm with @parvathyvr as she hasn't released the design doc. We discussed over her sketch file earlier this week.

**Comment by parvathyvr** [comment]
@divyanshiGupta @christianvogt Yes!this is correct..we will not have any filters in the Revision History page and Default sort should be revision with the latest revision at the top.

> @Divyansh42 @parvathyvr I think we agreed that there isn't any value to having name or status filters on this page because there's no value.
> 
> Default sort should be revision with the latest revision at the top.

**Comment by divyanshiGupta** [comment]
> @Divyansh42 yes but confirm with @parvathyvr as she hasn't released the design doc. We discussed over her sketch file earlier this week.

@parvathyvr confirmed that we dont need the filters.

**Comment by abhi-kn** [comment]
verified locally & works fine
/lgtm

**Pull request merged**